jerry-core/debugger/debugger.c

```diff
@@ -796,6 +796,20 @@ jerry_debugger_send_parse_function (uint32_t line, /**< line */
 } /* jerry_debugger_send_parse_function */
 
 /**
+ * Send the output of the program to the debugger client.
+ * Currently only sends print output.
+ */
+void
+jerry_debugger_send_output (jerry_char_t buffer[], jerry_size_t str_size, uint8_t subtype)
```

Comment:
subtype is a subbtype from the perspective of the debugger, but for the jerryscript API user this is the string type. So just call it type here.